openc3-cosmos-script-runner-api/app/models/running_script.rb

```diff
@@ -38,6 +38,17 @@
 SCRIPT_API = 'script-api'
 RUNNING_SCRIPTS = 'running-scripts'
 
+def running_script_publish(channel_name, data)
```

Comment:
Running script publish is here

Reply:
Why isn't it all anycable?

Reply:
The __anycable__ stream is for broadcasting out to clients.
We also use pub/sub for communicating with running scripts and that one isn't "anycable"